config/initializers/0_glitterconfig.rb

```diff
@@ -27,7 +27,13 @@
 # MAIL_PASSWORD=yourpassword
 # MAIL_AUTHENTICATION=plain
 Glitter::Application.config.action_mailer.delivery_method = :smtp
-Glitter::Application.config.action_mailer.smtp_settings={
+if Rails.env.development?
+	Glitter::Application.config.repo_dir="public/data"
+elsif Rails.env.test?
+	Glitter::Application.config.repo_dir="public/testdata"
+elsif Rails.env.production?
+	Glitter::Application.config.repo_dir="ENV['OPENSHIFT_DATA_DIR']/data"
+    Glitter::Application.config.action_mailer.smtp_settings={
```

Comment:
I can see an indentation mismatch in these lines. It's possible that the indentation wasn't correctly used before this PR, but this is a good time to fix it. We use 2 spaces (not tabs) for indentation.